app/views/signup/invited.html.erb

```diff
@@ -9,12 +9,6 @@
   <%= errors_for(@new_user) %>
 
   <div class="labelled_grid">
-    <% if not Rails.application.open_signups? %>
```

Comment:
Please don't delete this. We haven't had it enabled for ages, but new sister sites always turn it on at first.

Reply:
Ah, thanks for explaining, I was puzzled and guessed you were tidying apparently-dead code.

There's probably a lot of these sorts of minor hassles that I'm oblivious to because I almost always develop against a copy of the prod db and it's very easy for me to grab `Invitation.where(used_at: nil).last.token` or whatever in the rails console. If you see similar issues, please do post feature requests for things that would improve your developer experience.